aequilibrae/paths/graph.py

```diff
@@ -505,6 +512,71 @@ def save_compressed_correspondence(self, path, mode_name, mode_id):
         node_path = join(path, f"nodes_to_indices_c{mode_name}_{mode_id}.feather")
         pd.DataFrame(self.nodes_to_indices, columns=["node_index"]).to_feather(node_path)
 
+    def create_compressed_link_network_mapping(self):
```

Comment:
Doesn't this already exist in the the graph and network tables?  A   **__compressed_id__** field or something of that sort?

Reply:
This is a one-to-many map of compressed IDs to `link_id`s that can be accessed without using a groupby or other python related query.
For each compressed link ID, `idx[id]:idx[id + 1]` is the range of indices of `data` that contains the `link_id`s that the compressed link `id` corresponds to. It also sorts them in the correct manner

There is an existing mapping but I'm not sure exactly what it accomplishes, I think it maps graph link to network links. I don't believe it is for the compressed graph 
https://github.com/AequilibraE/aequilibrae/blob/6715319a6845be1de7ab015e074b963db84ba870/aequilibrae/paths/results/assignment_results.py#L251-L260